packages/transforms/filter/yaml-config.graphql

```diff
@@ -0,0 +1,3 @@
+extend type Transform {
+  filter: [String]
```

Comment:
Let's call it `filterSchema` because `filter` might sound like data-related. 
Also, It should be more complex, and allow either `whitelist` or `blacklist` to be set (but not both)